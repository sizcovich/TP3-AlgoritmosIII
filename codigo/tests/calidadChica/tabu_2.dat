15 15 28 1526600
25 25 48 4059300
35 35 68 9974000
45 45 88 19103600
55 55 108 32785500
65 65 128 12738000
75 75 148 24890200
85 85 168 42367900
95 95 188 66656200
105 105 208 97855200
