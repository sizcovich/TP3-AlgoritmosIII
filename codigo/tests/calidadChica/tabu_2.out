15 2 1 14 
25 2 1 6 
35 2 1 3 
45 2 1 44 
55 2 1 33 
65 2 1 17 
75 2 1 12 
85 2 1 25 
95 2 1 88 
105 2 1 55 
